src/Model/Carrier/CarrierDHLEuroPlus.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\Sdk\src\Model\Carrier;
+
+use MyParcelNL\Sdk\src\Model\Consignment\DHLEuroPlusConsignment;
+
+class CarrierDHLEuroPlus extends AbstractCarrier
+{
+    public const CONSIGNMENT = DHLEuroPlusConsignment::class;
+    public const HUMAN       = 'DHL Euro Plus';
+    public const ID          = 11;
+    public const NAME        = 'dhleuroplus';
+    public const TYPE        = 'b2b';
```

Comment:
Het beste is denk ik om van 'b2c' en 'b2b' constanten te maken ergens op een logische plek.